src/index.js

```diff
@@ -1,23 +1,19 @@
 var SentryCli = require('@sentry/cli');
 
-function SentryCliPlugin(options) {
-  options = options || {};
-  this.release = options.release;
-  this.include =
+function SentryCliPlugin(options = {}) {
```

Comment:
This change breaks the plugin on `node < 6`. I'm not sure what should be the resolution here. 
either avoid using ES6 default params or add node 6 to engines in package.json

Reply:
We'll lock it to >Node v6 in the next release.